librepcb/librepcb.pro

```diff
@@ -15,7 +15,7 @@ include(../common.pri)
 
 QT += core widgets opengl webkitwidgets xml printsupport sql
 
-exists(../.git):DEFINES += GIT_BRANCH=\\\"master\\\"
+exists(../.git):DEFINES += GIT_BRANCH=\\\"first_run_wizard\\\"
```

Comment:
haha :) At the beginning, I have used the line:
```
exists(.git):DEFINES += GIT_BRANCH=\\\"$$system(git rev-parse --abbrev-ref HEAD)\\\"
```
But on "some operating systems" this did not work, so I simply have replaced the command with a hardcoded string later...